generated-docs/Foreign/Class.md

```diff
@@ -71,9 +87,75 @@ Encode Char
 Encode Boolean
 Encode Number
 Encode Int
+(Encode a) => Encode (Identity a)
 (Encode a) => Encode (Array a)
 (Encode a) => Encode (Maybe a)
 (Encode v) => Encode (Object v)
+(RowToList r rl, EncodeRecord r rl) => Encode {  | r }
+```
+
+#### `DecodeWithOptions`
+
+``` purescript
+class DecodeWithOptions a  where
+  decodeWithOptions :: Options -> Foreign -> F a
+```
+
+When deriving `En`/`Decode` instances using `Generic`, we want
```

Comment:
@hdgarrood How's this?

Reply:
Looks good! I particularly like the class rename